src/context/BehandlingerContext.js

```diff
@@ -29,9 +31,39 @@ export const BehandlingerProvider = ({ children }) => {
         const valgtBehandling = behandlinger.find(
             b => b.behandlingsId === behandling.behandlingsId
         );
+
         setValgtBehandling(valgtBehandling);
     };
 
+    const fetchAlleBehandlinger = () => {
+        const fom = moment()
+            .subtract(1, 'days')
+            .format('YYYY-MM-DD');
+        const tom = moment().format('YYYY-MM-DD');
+        return behandlingerIPeriode(fom, tom)
+            .then(response => {
+                const newData = { behandlinger: response.data };
+                setBehandlinger(newData.behandlinger);
+                setValgtBehandling(undefined);
+                return;
```

Comment:
`return` her returnerer bare fra `then`-blokka, så den er ikke nødvendig og er sikkert lurt å fjerne for å minske muligheten for forvirring :)